layers/meta-resin-intel-quark/recipes-devtools/go/go-cross_%.bbappend

```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI += " \
+    file://0001-Revert-runtime-check-and-fail-early-with-a-message-i.patch \
```

Comment:
I'm wondering if these patches should not go in meta-resin and then only activate them if mmx is not available on the board. In this way multiple boards in the future will be able to benefit from it. @floion , what do you think?